homeassistant/components/foscam/camera.py

```diff
@@ -58,7 +58,11 @@ def __init__(self, device_info):
         self._foscam_session = FoscamCamera(
             ip_address, port, self._username, self._password, verbose=False)
 
-        self._media_port = self._foscam_session.get_port_info()[1]['mediaPort']
+        result, response = self._foscam_session.get_port_info()
+        if result == FOSCAM_COMM_ERROR:
```

Comment:
Where is `FOSCAM_COMM_ERROR` defined?  It's not imported here.  Also, might be cleaner to default `self._media_port` to None, and only set it if success, if there is an equivalent `FOSCAM_COMM_SUCCESS`
```
self._media_port = None
result, port_info = self._foscam_session.get_port_info()
if result == FOSCAM_COMM_SUCCESS:
    self._media_port = port_info['mediaPort']
```

Reply:
it was already defined here: https://github.com/home-assistant/home-assistant/blob/2f3ca66608573a24077f8dacbc6cfc6d17745a56/homeassistant/components/foscam/camera.py#L26

from what I can tell though, `0` is the "success" code.

Reply:
would you prefer if I did `if result == 0` instead?